emails/views.py

```diff
@@ -534,6 +534,10 @@ def _sns_message(message_json):
         # an external sender to a relay user
         pass
 
+    # if account flagged for abuse, early return
+    if user_profile.is_flagged:
+        return HttpResponse("Address is temporarily disabled.")
+
```

Comment:
suggestion (non-blocking): This should move above the check for reply from external sender to Relay user. If this is a malicious actor, they will be able to bypass this prevention by changing from plain emails to replies.

Reply:
Good suggestion! There is already a check to see if the user is flagged before a reply is sent:
https://github.com/mozilla/fx-private-relay/blob/1b4a66baabc0775a7d31490ce80bb7854b93652c/emails/views.py#L598-L602
Was added when I first added the abuse metrics.